spec/services/subscriptions/create_service_spec.rb

```diff
@@ -447,8 +447,7 @@
             let(:old_plan) { create(:plan, amount_cents: 100, organization:, pay_in_advance: false) }
 
             it 'enqueues a job to bill the existing subscription' do
-              create_service.call
-              expect(BillSubscriptionJob).to have_been_enqueued.at_least(1).times
+              expect { create_service.call }.to have_enqueued_job(BillSubscriptionJob)
```

Comment:
Did you edit these tests because they were failing with your change in the setup 🤔